pkg/scanners/scannerv4/scannerv4.go

```diff
@@ -0,0 +1,164 @@
+package scannerv4
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/google/go-containerregistry/pkg/authn"
+	"github.com/google/go-containerregistry/pkg/name"
+	v1 "github.com/stackrox/rox/generated/api/v1"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/env"
+	"github.com/stackrox/rox/pkg/logging"
+	"github.com/stackrox/rox/pkg/registries"
+	"github.com/stackrox/rox/pkg/scanners/types"
+	s4util "github.com/stackrox/rox/pkg/scannerv4"
```

Comment:
i'm not a big fan of `s4util` but I can see how my previous suggestion of `pkgscanner` may be confusing. I won't block the PR over this

Reply:
`pkgscanner` is fine by me, can change in the future if it gets confusion.